### ISSUE TITLE
Update gradle wrapper properties

### ISSUE THREAD

**Issue by marioshtika** [ionic-team/capacitor-configure#56]
Hello everyone.

Is there a way to change the gradle wrapper properties, found in the `gradle/wrapper/gradle-wrapper.properties` file, using the capacitor configure?

```
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-all.zip
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
```

I recently updated the gradle version in the `build.gradle` file and it is asking me to update the `distributionUrl`

from 
```
distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-all.zip
```
to
```
distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Thank you in advance.

**Comment by mlynch** [comment]
Started working on this, don't think it'll take much time to support

**Comment by marioshtika** [comment]
Thank you @mlynch 

Let me know if you need me to do anything, or test it when it is ready.

**Comment by mlynch** [comment]
Now available in 2.0.10. To use:

```yaml
platforms:
  android:
    properties:
       - file: gradle.properties
          entries:
            org.gradle.jvmargs: 'blah'
```

**Comment by marioshtika** [comment]
Hello @mlynch and sorry for the late response.

I am using `"@trapezedev/configure": "^3.0.7",` and I am getting the below message

```
run android properties 
[warn] Unsupported configuration option android.properties. Skipping
```

Is this not supported anymore?

Thank you in advance.
